MapCanvas.py

```diff
@@ -71,12 +76,12 @@ def parse_png_map(self, map_file_path):
 
         pixels_matrix = map_file.load()
 
```

Comment:
I don't know I guess there was something that didn't work so I try to change but anyway it didn't disturbing at all